solidity/random-beacon/tasks/mint.ts

```diff
@@ -0,0 +1,39 @@
+import { task, types } from "hardhat/config"
+
+import type { BigNumber } from "ethers"
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+
+task("mint", "Mints and approves T tokens for staking")
+  .addParam("owner", "Stake Owner address", undefined, types.string)
+  .addOptionalParam("amount", "Stake amount", 1_000_000, types.int)
+  .setAction(async (args, hre) => {
+    await setup(hre, args)
+  })
+
+async function setup(
+  hre: HardhatRuntimeEnvironment,
+  args: {
+    owner: string
+    amount: BigNumber
+  }
+) {
+  const { ethers, helpers } = hre
+  const { owner, amount } = args
+
+  const { deployer } = await helpers.signers.getNamedSigners()
+  const { to1e18 } = helpers.number
+  const t = await helpers.contracts.getContract("T")
+  const staking = await helpers.contracts.getContract("TokenStaking")
+
+  const toMintAndApprove = to1e18(amount)
+
+  console.log(
+    `Minting ${toMintAndApprove.toString()} T for the ${owner} and approving for staking ...`
```

Comment:
```suggestion
    `Minting ${toMintAndApprove.toString()} T for the ${owner} and approving for staking...`
```

Same comment to all other logs like this one across all files.

Reply:
[37a67eb](https://github.com/keep-network/keep-core/pull/3096/commits/37a67eb16efc6a6c63c958c7989a8c57293f2e65)